docs/modules/client.rst

```diff
@@ -0,0 +1,15 @@
+Client Connection
+=================
+
+The IPC client is very simple.
+It will simply connect to your server process and send JSON data.
+If you do not supply a port on initialisation, the client will connect to the multicast server
+(see the server section) and return the port from said server.
+If you do supply a port, it will connect to the server instantly.
+
+Requests are made by calling ``ipc.client.request(endpoint, **kwargs)``
+and will be sent to the server in the json format specified above.
+It will then wait for a response and return the data.
+
+.. autoclass:: discord.ext.ipc.client.Client
```

Comment:
Use the `currentmodule` directive.

Reply:
Please advise further? Using that directive I am either running into build errors or a failure to get any documentation outputted?
I haven't used it before, if you could provide an example that'd be beautiful. What I've found online hasn't been helpful so far

Reply:
The [`currentmodule` directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html?highlight=currentmodule#directive-py-currentmodule) should exist at the top of a file. The following should be valid:

```rst
.. currentmodule:: discord.ext.ipc.client

.. autoclass:: Client
```